jobs/config.json

```diff
@@ -11355,54 +11355,55 @@
       "sig-scheduling"
     ]
   },
-  "pull-kubernetes-e2e-gke-device-plugin-gpu": {
+  "pull-kubernetes-e2e-gke": {
+    "_comment1": "NOTE: THESE ARE THE JENKINS ARGS / COMMON ARGS (!)",
+    "_comment2": "PLEASE PUT ARGS THAT WILL WORK FOR 1.6 HERE",
+    "_comment3": "Args for 1.7+ should go in prow/config.yaml after the `--` !",
     "args": [
-      "--build=bazel",
+      "--build",
       "--cluster=",
       "--deployment=gke",
       "--extract=local",
       "--gcp-cloud-sdk=gs://cloud-sdk-testing/ci/staging",
       "--gcp-node-image=gci",
-      "--gcp-project=k8s-gke-gpu-pr",
-      "--gcp-zone=us-west1-b",
+      "--gcp-project=k8s-jkns-pr-gci-gke",
+      "--gcp-zone=us-central1-f",
       "--ginkgo-parallel=30",
-      "--gke-create-command=alpha container clusters create --quiet --enable-kubernetes-alpha --accelerator=type=nvidia-tesla-k80,count=2",
       "--gke-environment=test",
+      "--gke-shape={\"default\":{\"Nodes\":4,\"MachineType\":\"n1-standard-2\"}}",
+      "--mode=docker",
       "--provider=gke",
-      "--stage=gs://kubernetes-release-dev/ci",
-      "--stage-suffix=pull-gke-device-plugin-gpu",
-      "--test_args=--ginkgo.focus=\\[Feature:GPUDevicePlugin\\] --minStartupPods=8",
-      "--timeout=60m"
+      "--stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gke",
+      "--test_args=--ginkgo.flakeAttempts=2 --ginkgo.skip=\\[Slow\\]|\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\] --minStartupPods=8",
+      "--timeout=65m"
     ],
     "scenario": "kubernetes_e2e",
     "sigOwners": [
-      "sig-scheduling"
+      "sig-gcp"
     ]
   },
-  "pull-kubernetes-e2e-gke-gci": {
+  "pull-kubernetes-e2e-gke-device-plugin-gpu": {
     "args": [
-      "--build",
+      "--build=bazel",
       "--cluster=",
       "--deployment=gke",
       "--extract=local",
       "--gcp-cloud-sdk=gs://cloud-sdk-testing/ci/staging",
       "--gcp-node-image=gci",
-      "--gcp-project=k8s-jkns-pr-gci-gke",
-      "--gcp-zone=us-central1-f",
+      "--gcp-project=k8s-gke-gpu-pr",
+      "--gcp-zone=us-west1-b",
       "--ginkgo-parallel=30",
+      "--gke-create-command=alpha container clusters create --quiet --enable-kubernetes-alpha --accelerator=type=nvidia-tesla-k80,count=2",
       "--gke-environment=test",
-      "--gke-shape={\"default\":{\"Nodes\":4,\"MachineType\":\"n1-standard-2\"}}",
-      "--mode=docker",
       "--provider=gke",
-      "--runtime-config=batch/v2alpha1=true,admissionregistration.k8s.io/v1alpha1=true",
       "--stage=gs://kubernetes-release-dev/ci",
-      "--stage-suffix=pull-gke-gci",
-      "--test_args=--ginkgo.flakeAttempts=2 --ginkgo.skip=\\[Slow\\]|\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\] --minStartupPods=8",
-      "--timeout=55m"
+      "--stage-suffix=pull-gke-device-plugin-gpu",
```

Comment:
unify the --stage flag here as well

Reply:
done I think?

Reply:
as long as presubmit passes :-)

Reply:
Too many legacy flags 🤷‍♂️

Reply:
we should clean up the code, seems `--stage-suffix` is not used anywhere anymore :-)

Reply:
Will remove it in a follow up, who knows what jobs out there are trying to use this right now.